pkg/detectors/loggly/loggly_test.go

```diff
@@ -0,0 +1,161 @@
+//go:build detectors
+// +build detectors
+
+package loggly
+
+import (
+	"context"
+	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"testing"
+	"time"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+func TestLoggly_FromChunk(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors4")
+	if err != nil {
+		t.Fatalf("could not get test secrets from GCP: %s", err)
+	}
+	secret := testSecrets.MustGetField("LOGGLY")
+	inactiveSecret := testSecrets.MustGetField("LOGGLY_INACTIVE")
+
+	type args struct {
+		ctx    context.Context
+		data   []byte
+		verify bool
+	}
+	tests := []struct {
+		name                string
+		s                   Scanner
+		args                args
+		want                []detectors.Result
+		wantErr             bool
+		wantVerificationErr bool
+	}{
+		{
+			name: "found, verified",
+			s:    Scanner{},
+			args: args{
+				ctx:    context.Background(),
+				data:   []byte(fmt.Sprintf("You can find a loggly secret %s within", secret)),
```

Comment:
@ankushgoel27 https://github.com/trufflesecurity/trufflehog/pull/1782/files#r1336331681 the `data` only includes a `secret`. This detector has multi-part credentials and requires both a domain and secret to be verified. Does that make sense? Make sure you have verified _this_ test is passing please

Reply:
fixed the loggly_test.go